libcontainer/intelrdt/intelrdt.go

```diff
@@ -439,6 +439,22 @@ func getMemBwInfo() (*MemBwInfo, error) {
 	return memBwInfo, nil
 }
 
+// Get diagnostics for last filesystem operation error from file info/last_cmd_status
+func getLastCmdStatus() (string, error) {
+	rootPath, err := getIntelRdtRoot()
+	if err != nil {
+		return "", err
+	}
+
+	path := filepath.Join(rootPath, "info")
+	lastCmdStatus, err := getIntelRdtParamString(path, "last_cmd_status")
+	if err != nil {
+		return "", err
+	}
```

Comment:
What if the system doesn't support "last_cmd_status", is it gonna fall back to use the error we got in `getIntelRdtParamString`? Doesn't sound quite right.